### PR TITLE
OLM-Low-OCP-23670-Checking description info for csc CRD￼

### PR DESCRIPTION
**Pull request by tbuskey**
Test Case [OCP-23670](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-23670)  & part of Jira [OLM-1428](https://issues.redhat.com/browse/OLM-1428)

#### Does the following:
- oc explain csc _to check if the description is empty_

#### How to test:
`./extended-platform-tests run all --dry-run | grep 'Checking description info for csc CRD' | ./extended-platform-tests run -f -`

#### Successful run:
[testrun-23670.log](https://github.com/openshift/openshift-tests/files/4282967/testrun-23670.log)

> Mar  3 13:25:04.723: INFO: Running 'oc --kubeconfig=/home/tbuskey/working/44/config explain csc'
> [AfterEach] [Feature:Platform] an end user use OLM
> passed: (7.9s) 2020-03-03T18:25:06 "[Feature:Platform] an end user use OLM Checking description info for csc CRD￼ [Suite:openshift/conformance/parallel]"
> 1 pass, 0 skip (7.9s)

> oc version
> Client Version: openshift-clients-4.3-31-gc25fb9c7
> Server Version: 4.3.0-0.nightly-2020-03-03-052929
> Kubernetes Version: v1.16.2
>

### CODE REVIEW

**Comment by tbuskey** [comment]
@jianzhangbjz @bandrade @scolange  @kuiwang02 @mffiedler

File: test/extended/operators/olm.go [hunk 1]
```diff
@@ -227,27 +227,16 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
...
+		olmExplain := "csc"
+		msg, err := oc.AsAdmin().WithoutNamespace().Run("explain").Args(olmExplain).Output()
+		if err != nil {
+			e2e.Failf("Could not run oc explain %v:\n%v", olmExplain, msg)
 		}
-		if olmErr != 0 {
-			// fmt.Printf("explain errors: %d\n", olmErr)
-			e2e.Failf("%v errors in explaining the following OLM descriptors: %v", olmErr, olmErrDescriptor)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if strings.Contains(msg, "<empty>") {
```
Comment (jianzhangbjz) [inline test/extended/operators/olm.go]: We should check if the `csc` is the `CatalogSourceConfig` kind.

**Comment by tbuskey** [comment]
Closing to merge master and resubmitting PR

**Pull request closed without merging**